contrib/kind.sh

```diff
@@ -86,6 +89,10 @@ usage() {
     echo "-dl  | --ovn-loglevel-nbctld        Log config for nbctl daemon DEFAULT: '-vconsole:info'."
     echo "-ep  | --experimental-provider      Use an experimental OCI provider such as podman, instead of docker. DEFAULT: Disabled."
     echo "-eb  | --egress-gw-separate-bridge  The external gateway traffic uses a separate bridge."
+    echo "-lr  | --local-kind-registry        Configure kind to use a local docker registry rather than manually loading images"
+    echo "-dd  | --dns-domain                 Configure a custom dnsDomain for k8s services, Defaults to 'cluster.local'"
+    echo "-cn  | --cluster-name               Configure the kind cluster's name"
+    echo "-ric | --run-in-container           Configure the script to be run from a docker container, allowing it to still communcate with the kind controlplane" 
```

Comment:
what's the value/usecase for this?

Reply:
If a user runs the kind script within a container, the API server is no longer reachable at `localhost:6443`, this toggle swaps out `localhost` for the control-plane-node's IP in the kubeconfig 

```
# run_script_in_container should be used when kind.sh is run nested in a container 
# and makes sure the control-plane node is rechable by substituting 127.0.0.1 
# with the control-plane container's IP 
run_script_in_container() { 
  local master_ip=$(docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' ${KIND_CLUSTER_NAME}-control-plane | head -n 1)
  sed -i -- "s/server: .*/server: https:\/\/$master_ip:6443/g" $KUBECONFIG
  chmod a+r $KUBECONFIG
}
```